### ISSUE TITLE
Direct upload failed

### ISSUE THREAD

**Issue by smit1678** [hotosm/oam-catalog#99]
Browser based uploading failed. Looking at the logs on the server and seeing: 

```
4:53:09 PM web.1    |      {"id":"1501001589504:7c9e9422deb5:12:j3zmdkrv:17535","method":"post","url":"/uploads/url?access_token=TOKEN","agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/59.0.3071.115 Safari/537.36"}
4:53:09 PM web.1    |  Debug: response
```

<img width="1399" alt="screen shot 2017-07-25 at 6 03 04 pm" src="https://user-images.githubusercontent.com/796838/28584143-8c52d448-7163-11e7-9d14-15789d3a8f77.png">

Used this file as a test: https://www.dropbox.com/s/44vecb3z0ukje0e/Emae_Pakoro_Ortho_09042015_0800_jp2.jp2

**Comment by tombh** [comment]
I just pushed a commit to fix this in #228

**Comment by tombh** [comment]
Fixed in hotosm/oam-browser#228
